collector/workitem.go

```diff
@@ -0,0 +1,337 @@
+package collector
+
+import (
+	"context"
+	"fmt"
+
+	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/fabric8-services/fabric8-notification/wit"
+	"github.com/fabric8-services/fabric8-notification/wit/api"
+	goajwt "github.com/goadesign/goa/middleware/security/jwt"
+	"github.com/goadesign/goa/uuid"
+)
+
+func NewCommentResolver(c *api.Client) ReceiverResolver {
+	return func(ctx context.Context, id string) ([]Receiver, map[string]interface{}, error) {
+		cID, err := uuid.FromString(id)
+		if err != nil {
+			return []Receiver{}, nil, fmt.Errorf("Could not lookup comment based on id %v", id)
```

Comment:
Use lowercase for the error messages and avoid use of negation. It is better to use `unable to lookup comment based on id..`